dev/tasks/tasks.yml

```diff
@@ -940,6 +940,7 @@ tasks:
       - flight-sql-jdbc-driver-{no_rc_snapshot_version}-tests.jar
       - flight-sql-jdbc-driver-{no_rc_snapshot_version}.jar
       - flight-sql-jdbc-driver-{no_rc_snapshot_version}.pom
+      - arrow-bom-{no_rc_snapshot_version}.pom
```

Comment:
Could you keep this list in alphabetical order?

Reply:
Done.